.changeset/hip-tips-lay.md

```diff
@@ -0,0 +1,7 @@
+---
+"@pnpm/filter-workspace-packages": major
```

Comment:
Doesn't seem like you did breaking changes for this package. You just added a new option, so this should be minor

Reply:
done [here](https://github.com/pnpm/pnpm/pull/3035/commits/47ed18ad4218d113162d3008af414dc58cc90cf1)